cypress/e2e/app_commands/start.rb

```diff
@@ -0,0 +1,8 @@
+if defined?(DatabaseCleaner)
+  DatabaseCleaner.start
+else
+  msg = "XXX DatabaseCleaner not defined, can't start!!!"
+  logger.warn(msg)
+  raise msg
+end
+puts "DatabaseCleaner started"
```

Comment:
I'm debating removing this entirely and cache the max ids / latest rows once at startup, in the config/initializer when we configure the database cleaner strategy.  So, then tests would need only do:

```js
  afterEach(() => {
    cy.app('clean') // or 'db_clean' or whatever we want to call it
  };
```

There would be no need for a beforeEach() to cache the db rows.  The only reason to keep the beforeEach() is if we want to create rows and have them persist for some reason.

Reply:
There's also the [`before :run` hook](https://docs.cypress.io/api/node-events/before-run-api#__docusaurus_skipToContent_fallback) but you might not have access to the endpoint by that point